include/netinet/arp.h

```diff
@@ -65,7 +65,7 @@
 
 /* All ARP ioctls take a pointer to a struct arpreq as their parameter: */
 
-struct arpreq
+struct aligned_data(sizeof(FAR void *)) arpreq
```

Comment:
change to?
```suggestion
struct aligned_data(sizeof(uint32_t)) arpreq
```
since sizeof(void *) is 8 bytes on 64bit arch.

Reply:
The intention of this PR is to solve the issue of pointer misalignment. I think the performance may be better if the pointer is aligned on the 64-bit architecture.

Reply:
all field in arpreq isn't bigger than 32bit, it's waste memory to align with 64bit.

Reply:
cc @zhhyu7